test_outputs/snapshot_tests/cargo_semver_checks__snapshot_tests__multiple_ambiguous_package_name_definitions-output.snap

```diff
@@ -3,11 +3,14 @@ source: src/snapshot_tests.rs
 expression: result
 ---
 --- error ---
-package `example` is ambiguous: it is defined by in multiple manifests within the root path test_crates/manifest_tests/multiple_ambiguous_package_name_definitions
+Error while getting crate source from `RustdocFromProjectRoot`
```

Comment:
nit: our error messages always start with lowercase letters, because that makes them look better when listing multiple causes